src/lib/telemetry.js

```diff
@@ -16,6 +16,11 @@ const APP_NAME = 'debug-ping-view';
  * @returns {boolean} true if telemetry can be collected, else false.
  */
 function isTelemetryEnabled() {
+  // If the app is in development mode, we don't want to initialize Glean.
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
One question. Could `process.env.NODE_ENV` take any other value than `development` or `production`? e.g. could it ever be `undefined`? If yes then probably the following check might be better:

`if (process.env.NODE_ENV !== 'production') {
`

Reply:
I actually updated this to use the same check that other places in the app use, which is
`REACT_APP_ENV !== "prod"`.

This is how we determine which firebase config to use as well.